### DISCUSSION TITLE
Entrega tp0

### DISCUSSION THREAD

**Discussion by agusscarava** [INGCOM-UNRN-P1/cursada-2023#17] [Preguntas y respuestas] [answered]
Hola grupo !
les consulto una cosita, si ya modifique el archivo de formulario y agregue al repo la version del compilador ya estaria completa esta actividad ??

Gracias!

**Comment by martinvilu** [comment, marked as answer]
Estimaría que sí. Son dos archivos, el que contiene la salida de `gcc --version` y la "ficha"

**Reply by agusscarava** [reply]
Graciass!
